roles/proftpd/tasks/users.yml

```diff
@@ -5,3 +5,12 @@
     src:  "{{ manala_proftpd_users_template|ternary(manala_proftpd_users_template, 'users/default.j2') }}"
     dest: "{{ manala_proftpd_users_file }}"
     mode: 0600
+
+- name: users > home directories
+  file:
+    path: "{{ item.home }}"
+    owner: manala_proftpd_users_default_uid
```

Comment:
should be `user.uid|default(manala_proftpd_users_default_uid)` i guess

Reply:
Done ! thx